code/espurna/sensor.cpp

```diff
@@ -911,6 +932,11 @@ const char * const _magnitudeSettingsPrefix(unsigned char type) {
     case MAGNITUDE_PM2dot5: return "pm1dot5";
     case MAGNITUDE_PM10: return "pm10";
     case MAGNITUDE_CO2: return "co2";
+    case MAGNITUDE_CO2E: return "co2e";
+    case MAGNITUDE_BVOC: return "bvoc";
+    case MAGNITUDE_IAQ: return "iaq";
+    case MAGNITUDE_IAQ_ACCURACY: return "iaqAccuracy";
+    case MAGNITUDE_STATIC_IAQ: return "staticIaq";
```

Comment:
@mcspr should these be camelCase?

Reply:
Yep. Lowercase abbreviations seem easier to read.
I just wonder if staticIaq should be iaqStatic instead

Reply:
I prefer `iaqStatic`.. better alignment.